commit 8c8a71c85f2ed5cc90d08d82ce645513fc907cb6
Author: Siddhesh Poyarekar <siddhesh@sourceware.org>
Date:   Mon Jan 24 10:57:09 2022 +0530

    tst-realpath-toolong: Fix hurd build
    
    Define PATH_MAX to a constant if it isn't already defined, like in hurd.
    
    Signed-off-by: Siddhesh Poyarekar <siddhesh@sourceware.org>
    (cherry picked from commit 976db046bc3a3738f69255ae00b0a09b8e77fd9c)

diff --git a/stdlib/tst-realpath-toolong.c b/stdlib/tst-realpath-toolong.c
index 8bed772460b37571..4388890294374601 100644
--- a/stdlib/tst-realpath-toolong.c
+++ b/stdlib/tst-realpath-toolong.c
@@ -29,6 +29,10 @@
 
 #define BASENAME "tst-realpath-toolong."
 
+#ifndef PATH_MAX
+# define PATH_MAX 1024
+#endif
+
 int
 do_test (void)
 {
